src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface IPagination {
+  perPage: number;
+  total: number;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  getCurrentItems: () => string[];
+}
+
+export const Pagination = ({ page, setPage, perPage, total, getCurrentItems }: IPagination) => {
+
+  // Número total de páginas
+  const listNumbers = Math.ceil(total / perPage);
+
+  // Geração dos números de páginas
+  const pageNumbers = getNumbers(1, listNumbers);
```

Comment:
The usage of 'getNumbers' function is not shown in this snippet. Ensure that it is correctly implemented to generate an array of page numbers. If the utility function is complex or not self-explanatory, consider adding a comment explaining its purpose.